components/header.tsx

```diff
@@ -39,7 +39,7 @@ export default function Header() {
   const [isOpen, setIsOpen] = useState(false)
 
   const toggleMenu = () => {
-    setIsOpen(!isOpen)
```

Comment:
Is this change really necessary? It was more compact and more intuitive before the change.

Reply:
Also, this is out of scope for the issue that is being solved. It is better to submit separate PRs for separate issues.

Reply:
Yeah, that wasn't really related to the issue, but it was a small change, so I added it. I'll be more careful next time. Thanks for the feedback—another lesson learned in open source☺️!

## Why I had made change?

Reply:
Change Updated!